packages/workflow/src/TelemetryHelpers.ts

```diff
@@ -129,106 +107,133 @@ export function generateNodesGraph(
-
-				nodeItem.credential_type = {
-					none: 'none',
-					genericCredentialType: node.parameters.genericAuthType as string,
-					predefinedCredentialType: node.parameters.nodeCredentialType as string,
-				}[authentication];
-
-				nodeItem.credential_set = node.credentials
-					? Object.keys(node.credentials).length > 0
-					: false;
-
-				const { url } = node.parameters as { url: string };
-
-				nodeItem.domain_base = getDomainBase(url);
-				nodeItem.domain_path = getDomainPath(url);
-				nodeItem.method = node.parameters.requestMethod as string;
-			} else if (node.type === 'n8n-nodes-base.webhook') {
-				webhookNodeNames.push(node.name);
-			} else {
+		if (node.type === 'n8n-nodes-base.httpRequest' && node.typeVersion === 1) {
+			try {
+				nodeItem.domain = new URL(node.parameters.url as string).hostname;
+			} catch {
+				nodeItem.domain = getDomainBase(node.parameters.url as string);
+			}
+		} else if (node.type === 'n8n-nodes-base.httpRequest' && node.typeVersion > 1) {
+			const { authentication } = node.parameters as { authentication: string };
+
+			nodeItem.credential_type = {
+				none: 'none',
+				genericCredentialType: node.parameters.genericAuthType as string,
+				predefinedCredentialType: node.parameters.nodeCredentialType as string,
+			}[authentication];
+
+			nodeItem.credential_set = node.credentials ? Object.keys(node.credentials).length > 0 : false;
+
+			const { url } = node.parameters as { url: string };
+
+			nodeItem.domain_base = getDomainBase(url);
+			nodeItem.domain_path = getDomainPath(url);
+			nodeItem.method = node.parameters.requestMethod as string;
+		} else if (node.type === 'n8n-nodes-base.webhook') {
+			webhookNodeNames.push(node.name);
+		} else {
+			try {
 				const nodeType = nodeTypes.getByNameAndVersion(node.type);
-
-				nodeType?.description?.properties?.forEach((property) => {
-					if (
-						property.name === 'operation' ||
-						property.name === 'resource' ||
-						property.name === 'mode'
-					) {
-						nodeItem[property.name] = property.default ? property.default.toString() : undefined;
+				if (nodeType) {
+					const nodeParameters = getNodeParameters(
+						nodeType.description.properties,
+						node.parameters,
+						true,
+						false,
+						node,
+					);
+
+					if (nodeParameters) {
+						const keys: Array<'operation' | 'resource' | 'mode'> = [
+							'operation',
+							'resource',
+							'mode',
+						];
+						keys.forEach((key) => {
+							if (nodeParameters.hasOwnProperty(key)) {
+								nodeItem[key] = nodeParameters[key]?.toString();
+							}
+						});
 					}
-				});
-
-				nodeItem.operation = node.parameters.operation?.toString() ?? nodeItem.operation;
-				nodeItem.resource = node.parameters.resource?.toString() ?? nodeItem.resource;
-				nodeItem.mode = node.parameters.mode?.toString() ?? nodeItem.mode;
+				}
+			} catch (e: unknown) {
+				if (!(e instanceof Error && e.message.includes('Unrecognized node type'))) {
```

Comment:
not perfect. will follow up on this PR to try and standarize this somehow https://github.com/n8n-io/n8n/pull/8264